package-lock.json

```diff
@@ -20988,16 +20988,16 @@
     "packages/browserslist-config": {
       "name": "@decidim/browserslist-config",
       "version": "0.30.0-dev",
-      "license": "AGPL-3.0"
+      "license": "AGPL-3.0-or-later"
     },
     "packages/core": {
       "name": "@decidim/core",
       "version": "0.30.0-dev",
-      "license": "AGPL-3.0",
+      "license": "AGPL-3.0-or-later",
       "dependencies": {
         "@emoji-mart/data": "^1.1.2",
         "@rails/activestorage": "^7.0.8",
-        "@tarekraafat/autocomplete.js": "^10.2.6",
+        "@tarekraafat/autocomplete.js": "10.2.7",
```

Comment:
This was done automatically by running "npm install" 

(Just a heads-up, not sure if this is intentional - according to #13555 it's ok, but just in case)